build_library/prod_image_util.sh

```diff
@@ -102,7 +103,13 @@ create_prod_image() {
   fi
 
   tar -cf "${BUILD_DIR}/${image_pkgdb}" -C "${root_fs_dir}" var/cache/edb var/db/pkg
-
+  sudo cp -r "${root_fs_dir}" "${BUILD_DIR}/root_fs_dir2" 
+  sudo rsync -a --delete  "${BUILD_DIR}/configroot/etc/portage" "${BUILD_DIR}/root_fs_dir2/etc" 
+  mksquashfs "${BUILD_DIR}/root_fs_dir2"  "${BUILD_DIR}/flatcar_production_image_sysext.squashfs" -noappend
+  sudo rm -rf "${BUILD_DIR}/root_fs_dir2"
+  # Run build_sysext script to create python sysext so we can see it from GitHub actions CI workflow
+  sudo "$(dirname ${BASH_SOURCE[0]})/../build_sysext" --board="${BOARD}" python-sysext python
```

Comment:
```suggestion
  sudo "$(dirname ${BASH_SOURCE[0]})/../build_sysext" --board="${BOARD}" --group="${FLAGS_group}" --output_root="${FLAGS_output_root}" --build_attempt="${FLAGS_build_attempt}" python-sysext python
```
I think we need to align these values